rest/plugins/fqdnblocklist.py

```diff
@@ -205,18 +205,18 @@ def onMessage(self, request, response):
         try:
             for i in request.json:
                 # were we checked?
-                if self.name in i:
-                    blockfqdn = i.values()[0]
-                if 'fqdn' in i:
-                    fqdn = i.values()[0]
-                if 'duration' in i:
-                    duration = i.values()[0]
-                if 'comment' in i:
-                    comment = i.values()[0]
-                if 'referenceid' in i:
-                    referenceID = i.values()[0]
-                if 'userid' in i:
-                    userid = i.values()[0]
+                if self.name in list(i.keys()):
```

Comment:
These changes aren’t needed, you can look for if a key exists in python3 the same as python2

Reply:
removed .keys() from the fqdnblocklist.py